LeaderboardBackend/Controllers/RunsController.cs

```diff
@@ -51,4 +53,24 @@ public async Task<ActionResult> CreateRun([FromBody] CreateRunRequest request)
 		await RunService.CreateRun(run);
 		return CreatedAtAction(nameof(GetRun), new { id = run.Id }, run);
 	}
+
+	/// <summary>Gets the Participations for a Run.</summary>
+	/// <param name="id">The Run ID.</param>
+	/// <response code="200">An array with all participations. Can be empty</response>
+	/// <response code="404">If the run is not found.</response>
```

Comment:
Sorry for this change, but I'd like for empty participations to also return a 404. Also type entities in lowercase as well, please :pray: 
```suggestion

	/// <summary>Gets the participations for a run.</summary>
	/// <param name="id">The run ID.</param>
	/// <response code="200">An array with all participations.</response>
	/// <response code="404">If the run or no participations are found.</response>
```